tests/models/test_evaluation.py

```diff
@@ -176,6 +176,10 @@ def convert_num_to_label(x):
 
 @pytest.fixture
 def pipeline_model_uri():
+    return get_pipeline_modlel_uri()
+
+
+def get_pipeline_modlel_uri():
```

Comment:
Do we need to extract this into a separate method that can be called without a fixture? Because the method doesn't take any arguments, it doesn't seem necessary; test cases can just use the fixture.

Reply:
This one I am kinda stuck because I put those into functions is so that the parameterized `baseline_model_uri` fixture need to call these models to get different models; pytest does not allow call fixture directly; hence I made those into functions to be callable

Reply:
Got it - makes sense! 

```suggestion
def get_pipeline_model_uri():
```